Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/Inputs/SEC101_026.MailgunApiCredentials.ps1

```diff
@@ -1,5 +1,5 @@
 # AppSettings style
 <appSettings>
   <add key="MailgunDomain" value="deadbeef0deadbeef0deadbeef.mailgun.org" />
-  <add key="MailgunApiKey" value="0deadbeef00deadbeef00deadbeef000-deadbeef-deadbeef" />
+  <add key="MailgunApiCredentials" value="0deadbeef00deadbeef00deadbeef000-deadbeef-deadbeef" />
```

Comment:
The difference between "Key" and "Credentials" is purely internal, right?  Input files should not change, if I'm not mistaken.